src/lib/apis/evaluator.ts

```
@@ -3,8 +3,8 @@ import { ExtractAPIDataType } from "../utils/utils";
 import { MaximAPI } from "./maxim";
 
 export class MaximEvaluatorAPI extends MaximAPI {
-	constructor(baseUrl: string, apiKey: string) {
-		super(baseUrl, apiKey);
+	constructor(baseUrl: string, apiKey: string, isDebug?: boolean) {
+		super(baseUrl, apiKey, isDebug);
 	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)